front/app/modules/commercial/idea_custom_fields/admin/containers/projects/edit/ideaform/IdeaCustomField.tsx

```diff
@@ -233,103 +126,86 @@ export default memo<Props>(
-            unmountOnExit={true}
-            enter={true}
-            exit={true}
-          >
-            <CollapseContainer>
-              <CollapseContainerInner>
-                <Toggles>
-                  {canSetEnabled && (
-                    <ToggleContainer>
-                      <StyledToggle
-                        checked={fieldEnabled}
-                        onChange={handleEnabledOnChange}
-                        label={<FormattedMessage {...messages.enabled} />}
-                        labelTextColor={colors.adminTextColor}
-                        className={`
-                        e2e-${ideaCustomField.attributes.code}-enabled-toggle-label
-                      `}
-                      />
-                      <IconTooltip
-                        content={
-                          <FormattedMessage
-                            {...messages.enabledTooltipContent}
-                          />
-                        }
-                      />
-                    </ToggleContainer>
-                  )}
-                  {fieldEnabled && canSetRequired && (
-                    <ToggleContainer>
-                      <StyledToggle
-                        checked={fieldRequired}
-                        onChange={handleRequiredOnChange}
-                        label={<FormattedMessage {...messages.required} />}
-                        labelTextColor={colors.adminTextColor}
-                        className={`
-                          e2e-${ideaCustomField.attributes.code}-required-toggle-label
-                      `}
-                      />
-                      <IconTooltip
-                        content={
-                          <FormattedMessage
-                            {...messages.requiredTooltipContent}
-                          />
-                        }
-                      />
-                    </ToggleContainer>
-                  )}
-                </Toggles>
-
-                {fieldEnabled && (
-                  <Suspense fallback={<Spinner />}>
-                    <QuillMutilocWithLocaleSwitcher
-                      id={`${ideaCustomField.id}-description`}
-                      noImages={true}
-                      noVideos={true}
-                      noAlign={true}
-                      valueMultiloc={descriptionMultiloc}
-                      onChange={handleDescriptionOnChange}
-                      label={
-                        <LocaleSwitcherLabelText>
-                          <FormattedMessage {...messages.descriptionLabel} />
-                        </LocaleSwitcherLabelText>
-                      }
-                    />
-                  </Suspense>
-                )}
-              </CollapseContainerInner>
-            </CollapseContainer>
-          </CSSTransition>
-        </Container>
+        <Accordion
+          title={
+            <Box
+              onMouseDown={removeFocusAfterMouseClick}
+              className={`
```

Comment:
Could you check that the e2e test this refers to still passes?
You can use `npm run cypress:open` to run it and check.

Reply:
I already ran the tests that are attached to this and they were working. Is it failing on your end?

Reply:
No, I didn't check. Did you run the cypress tests, as well, or just the regular unit tests?